### DISCUSSION TITLE
Is there a way to convert string to a type variable

### DISCUSSION THREAD

**Discussion by tushargugnani** [11ty/eleventy#3041] [Q&A] [answered]
1. Lets say I have a string in my template which evaluates to a string. (for example test variable is equal to 'foo')
2. I want to fetch data from foo.json file from the global data.
3. {{test.name}} does not work since test is current a string
4. How can I change a string to of type variable.

**Comment by pdehaan** [comment, marked as answer]
I've found it easiest to nest your global data files in a subdirectory.
For example: ./src/_data/users/foo.json

Then you should be able to do something like this:

```njk
---
test: foo
---

{{ users[test].name }}
```

**Reply by tushargugnani** [reply]
Awesome, Thanks a lot. 😊
